### PR TITLE
Prepare a debug action

### PR DESCRIPTION
**Pull request by bripkens**

### CODE REVIEW

**Comment by basti1302** [comment]
@bripkens do we still need this or can it be closed?

**Comment by bripkens** [comment]
@basti1302: I didn't get to continue this. The idea here was to support backchannel messages which would gather all the typically requested debugging data. In this case specifically around loaded modules, e.g. so that these may be requested via the backchannel. There is currently no need for it, which is why I didn't continue with it. This being said, I still think that this will be valuable in the future.

File: test/actions/source_test.js [hunk 1]
```diff
@@ -9,13 +9,13 @@ var config = require('../config');
 var utils = require('../utils');
 
 
-describe('actions/source', function() {
+describe.only('actions/source', function() {
```
Comment (basti1302) [inline test/actions/source_test.js]: forgotten only

Comment (bripkens) [reply test/actions/source_test.js]: Ah, damn it. We really need a lint step for this…

**Pull request merged**